src/ts/greedy-nav/GreedyNav.test.ts

```diff
@@ -312,4 +313,58 @@ describe('Greedy Nav', () => {
       expect(nav.navDropdownToggle!.innerHTML).toContain('More');
     });
   });
+
+  describe('menu opening and closing', () => {
+    let dom: JSDOM;
+    let document: HTMLDocument;
+    let nav: GreedyNavMenu;
+
+    beforeEach(async () => {
+      dom = await JSDOM.fromFile(
+        path.join(__dirname, './__fixtures__/menu.html'),
+        { url: 'http://www.example.com/menu.html' }
+      );
+
+      document = dom.window.document;
+
+      nav = new GreedyNavMenu(defaultConfig, document);
+      nav.init();
+    });
+
+    afterEach(() => {
+      document.body.innerHTML = '';
+    });
+
+    describe('openDropDown', () => {
+      it('opens the dropdown menu', () => {
+        nav.openDropDown(nav.mainNavWrapper);
```

Comment:
Any reason not to trigger this the same way the user does (via click or keyboard press)?